stim.proto

```diff
@@ -0,0 +1,571 @@
+syntax = "proto3";
+package openmind;
+
+option csharp_namespace = "OpenMind";
+
+import "summit.proto";
+
+service StimService {
```

Comment:
Weren't we going to put the stim commands in the 'device' service as opposed to creating a new one?

Reply:
That's my recollection too!

Reply:
Sorry for the delay. I just pushed changes for this. I deleted stim.proto and moved everything into device.proto. Now I have a minor design question: device.proto now has two services (DeviceManagerService and StimService). Previously all the message types were loosely written in order form highest level to lowest level. But the crude way I pasted in StimService breaks that (although within StimService, that order is preserved). Is that okay? Should I move the services to the top and re-order the message types? Does it matter? 

If that didn't make sense, we can talk about it in the meeting tomorrow.